rasterio/warp.py

```diff
@@ -1,22 +1,19 @@
 """Raster warping and reprojection."""
 
 
-from __future__ import absolute_import
-from __future__ import division
+from __future__ import absolute_import, division
```

Comment:
Looks like my editor autosorted the imports with isort. If you would prefer it to be how it was before, just let me know and I can undo it.